### PR TITLE
Add Homebrew to README.md

### PR DESCRIPTION
**Pull request by umireon**
There is already a formula in Homebrew.
https://formulae.brew.sh/formula/cloudformation-guard
macOS users can install cloudformation-guard with brew in a handy way.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by omkhegde — APPROVED** [review]

**Review by dchakrav-github — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -162,6 +162,14 @@ $ curl --proto '=https' --tlsv1.2 -sSf https://raw.githubusercontent.com/aws-clo
 ```
 Remember to add `~/.guard/bin/` to your `$PATH`.
 
+Altenatively, you can install the latest version with [Homebrew](https://brew.sh/).
```
Comment (priyap286) [inline README.md:165]: Please correct the spelling - "Alternatively".

File: README.md [hunk 2]
```diff
@@ -162,6 +162,14 @@ $ curl --proto '=https' --tlsv1.2 -sSf https://raw.githubusercontent.com/aws-clo
...
+
+```bash
+$ brew install cloudformation-guard
+```
+
+In this way, you need not to modify your `$PATH`.
```
Comment (priyap286) [inline README.md:171]: Can you please change this line to, 
"You would not need to modify \`$PATH\` this way."

File: README.md (hunk 1 above)
Comment (umireon) [reply README.md]: Fixed. Thanks 🙇

File: README.md (hunk 2 above)
Comment (umireon) [reply README.md]: Changed. Thanks 🙇

**Comment by umireon** [comment]
@priyap286 I've made the requested changes.

**Pull request merged**